python/pyspark/sql/functions.py

```diff
@@ -426,6 +426,29 @@ def monotonically_increasing_id():
     return Column(sc._jvm.functions.monotonically_increasing_id())
 
 
+@since(2.1)
+def monotonically_increasing_id(offset):
```

Comment:
Or we can default offset to 0 which covers the current usage.
But I am not sure which version to put in @since()